app/code/Magento/InventorySales/etc/di.xml

```diff
@@ -29,6 +29,9 @@
         <plugin name="process_register_products_sale" type="Magento\InventorySales\Plugin\CatalogInventory\StockManagement\ProcessRegisterProductsSalePlugin"/>
         <plugin name="process_revert_products_sale" type="Magento\InventorySales\Plugin\CatalogInventory\StockManagement\ProcessRevertProductsSalePlugin"/>
     </type>
+    <type name="Magento\CatalogInventory\Model\Spi\StockStateProviderInterface">
+        <plugin name="check_quote_item_qty" type="Magento\InventorySales\Plugin\StockState\CheckQuoteItemQtyPlugin"/>
```

Comment:
why do we pluginize SPI interface (\Magento\CatalogInventory\Model\Spi\StockStateProviderInterface) but not 
\Magento\CatalogInventory\Api\StockStateInterface ? 

if we want to use suppress initial execution it makes sense to add plugin on API